aom/src/main/java/com/nedap/archie/adl14/ADL14DefaultMultiplicitiesSetter.java

```diff
@@ -9,10 +9,10 @@
 import com.nedap.archie.rminfo.MetaModels;
 
 /**
- * Sets the default occurrences with ADL 1.4 rules, if not explicitly set in a given Archetype.
- * Useful for conversion to ADL 2, where the default values are different, and it is good to start
+ * Sets the default occurrences with ADL 1.4 rules ({1..1}), if not explicitly set in a given Archetype.
+ * Useful for conversion to ADL 2, where the default values are different ({0..*}), and it is good to start
```

Comment:
they are not {0..*} default - they are whatever is in the reference model for that property. Which can be one of at least 0..1, 1..1, 0..* or 1..*.